ui/src/app/material/material.module.ts

```diff
@@ -38,7 +39,7 @@ const materialModules = [
   MatTableModule,
   MatExpansionModule,
   MatChipsModule,
-  MatTableModule,
```

Comment:
Why are we removing MatTableModule?

Reply:
It was a duplicate, its already declared in line 38.